osu.Game/Screens/Select/Carousel/CarouselItem.cs

```diff
@@ -15,6 +15,8 @@ public abstract class CarouselItem : IComparable<CarouselItem>
         /// </summary>
         public float CarouselYPosition;
 
+        public float ExpectedYPosition;
```

Comment:
what is `ExpectedYPosition` and how does it differ from `CarouselYPosition`? would like an xmldoc on the former and maybe a rename of the latter that explains the difference better.